opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java

```diff
@@ -105,6 +108,8 @@
 
     private String selectedFeatureInterventionType;
 
+    private String selectedFeatureTaskStatus;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'selectedFeatureTaskStatus' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4647909)